.travis.yml

```diff
@@ -1,13 +1,37 @@
 language: python
+
 python:
```

Comment:
I _think_ we need to remove the `python` stanza here if we're specifying `python` versions below in `matrix.include`.
I believe that's what's causing the two Travis-CI failures right now, since `TOXENV` isn't set there.

Reply:
Ah, I'll try that.